### DISCUSSION TITLE
Why can the official warehouse automatically load and initialize the module database, but the azerothcore warehouse of the playbot branch cannot?

### DISCUSSION THREAD

**Discussion by xengine-qyt** [mod-playerbots/mod-playerbots#589] [General]
why is that

**Comment by noisiver** [comment]
I'm sorry but I don't understand. What exactly are you asking?

**Reply by xengine-qyt** [reply]
I am using the Git version provided by AzerothCore. When using a module, the database files within the module are automatically imported into my database. However, when using the PlayBot branch, the module's data needs to be imported manually. Why is this happening? Why is there such a difference?
